Frontend/android-native/mobility-app/src/main/res/values-ta/strings.xml

```diff
@@ -76,4 +76,6 @@
     <string name="no_change">மாற்றம் இல்லை</string>
     <string name="no_additional_fare_tips_and_waiting_charges">"கூடுதல் கட்டணம் இல்லை. காலிகைகள் இல்லை. காதிரி கட்டணங்கள் இல்லை."</string>
     <string name="third_party_booking">"மூன்றாம் தரப்பு முன்பதிவு"</string>
+    <string name="parking_charges">%1$f₹ பார்க்கிங்</string>
```

Comment:
don't we need to round off the float value here . Shouldn't it be  `%1$.0f₹` instead of `%1$f₹`

Reply:
Please check the same for other languages as well.